custom_models/import_models/llama-3/llama3-sftt-llama3-fine-tuning.ipynb

```diff
@@ -0,0 +1,664 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "e495f494-6da2-451a-9bc3-9f04776c2c96",
+   "metadata": {},
+   "source": [
+    "In this notebook we will walk through how to fine-tune a Llama-3 LLM on Amazon SageMaker using PyTorch FSDP and Flash Attention 2 including Q-LORA and PEFT. This notebook also explains using PEFT and merging the adapters. \n",
```

Comment:
can you please add the sections from this link https://docs.aws.amazon.com/bedrock/latest/userguide/model-customization-import-model.html

Reply:
Added.